ApolloExtra.lua

```diff
@@ -2418,48 +2473,115 @@ OlympusLootData.ApolloUpgrade = {
+				{ Cue = "/VO/Apollo_0065",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Well, I have talked to my daughters, and they kind of don't want, hmm, a distant relationship, and since I don't see you going anywhere, I think they'll end this here." },
+			},
+			ApolloAboutHydra =
+			{
+                Name = "ApolloAboutHydra",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredBiome = "Asphodel",
+				RequiredAnyRoomsLastRun = { "B_Boss01", "B_Boss02" },
 				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloGift01", "ApolloFirstPickUp" },
-				RequiredGodLoot = "AresUpgrade",
-				{ Cue = "/VO/Apollo_0046",
+				{ Cue = "/VO/Apollo_0165",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "I sense the mark of my Lord Ares on you, love. He's poorly understood among the gods, yet surely you can see fair qualities in him, and me?" },
+					Text = "Hydra has snake's head, right? um, I hate snakes, specially pythons, they are the worst. Still, I got the upper hand against them, here that should help." },
+			},
+			ApolloAboutDaphne01 =
+			{
+                Name = "ApolloAboutDaphne01",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloGift04", "ApolloFirstPickUp" },
+				{ Cue = "/VO/Apollo_0162",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Were you looking at my laurel crown, Zagz... Oh Daphne, I miss you so much... Where are you..." },
+			},
+			ApolloAboutPan01 =
+			{
+                Name = "ApolloAboutPan01",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloGift02", "ApolloFirstPickUp" },
+				{ Cue = "/VO/Apollo_0163",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "As you know, Zagzag, I am the God of Music, but once the Nymph Pan tried to get my title, but he had no chance against my great taste in music. I still remember his face. Priceless..." },
```

Comment:
"As you know, Zagzag, I am the God of Music. But once the Nymph Pan tried to get my title, he had no chance against my great taste in music. I still remember his face. Priceless!"